src/main/java/de/srendi/advancedperipherals/common/util/LuaConverter.java

```diff
@@ -31,29 +38,64 @@
 
 public class LuaConverter {
 
-    public static Map<String, Object> entityToLua(Entity entity) {
+    public static Map<String, Object> entityToLua(Entity entity, boolean detailed) {
```

Comment:
Is it intended that here are entity related changes in the PR?

Reply:
yes cause the turtle need to get more details of its rider